.env.development

```diff
@@ -1,2 +1,7 @@
 REACT_APP_API_STUDY_SERVER=http://localhost:5001
 REACT_APP_API_CASE_SERVER=http://localhost:5000
+
```

Comment:
We have to implement a mode for the app where the authentication is mocked and we can login without an identityProvider. And this no-identityprovider mode It should be the default in development. Please try to implement it